lib/payment.js

```diff
@@ -227,9 +253,11 @@ class Payment {
             try {
                 const senderData = JSON.parse(jsonSenderData);
                 _senderRef.set(p, senderData.ref);
+                if (null !== senderData.payload && undefined !== senderData.payload)
```

Comment:
Can be simplified to 'if (! senderData)'

Reply:
Adjusted according to https://github.com/hubiinetwork/nahmii-sdk/pull/161#discussion_r528580919_.